### PR TITLE
1.9.6

### PR DESCRIPTION
**Pull request by VicV**
Volley dependency re-added because thats how we do network things..

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,7 +46,8 @@ dependencies {
   provided 'com.segment.analytics.android:analytics:4.0.0'
 
   //Taplytics
-  compile('com.taplytics.sdk:taplytics:1.7.0')
+  compile('com.taplytics.sdk:taplytics:1.9.6')
+  compile 'com.android.volley:volley:+'
```
Comment (f2prateek) [inline build.gradle]: We should pin a version here. I'm not sure what version Taplytics needs though.

Comment (VicV) [inline build.gradle]: @f2prateek yeah just stick with 1.0.1 they rarely update anyways.

**Comment by f2prateek** [comment]
I think this needs a custom maven repository? Build fails locally with `Could not find com.android.volley:volley:1.0.1` 

```
* What went wrong:
A problem occurred configuring root project 'analytics-android-integration-taplytics'.
> Failed to notify project evaluation listener.
   > Could not resolve all dependencies for configuration ':_debugCompile'.
      > Could not find com.android.volley:volley:1.0.1.
        Searched in the following locations:
            https://repo1.maven.org/maven2/com/android/volley/volley/1.0.1/volley-1.0.1.pom
            https://repo1.maven.org/maven2/com/android/volley/volley/1.0.1/volley-1.0.1.jar
            https://github.com/taplytics/Taplytics-Android-SDK/raw/master/AndroidStudio/com/android/volley/volley/1.0.1/volley-1.0.1.pom
            https://github.com/taplytics/Taplytics-Android-SDK/raw/master/AndroidStudio/com/android/volley/volley/1.0.1/volley-1.0.1.jar
            file:/Users/prateek/android-sdk/extras/android/m2repository/com/android/volley/volley/1.0.1/volley-1.0.1.pom
            file:/Users/prateek/android-sdk/extras/android/m2repository/com/android/volley/volley/1.0.1/volley-1.0.1.jar
            file:/Users/prateek/android-sdk/extras/google/m2repository/com/android/volley/volley/1.0.1/volley-1.0.1.pom
            file:/Users/prateek/android-sdk/extras/google/m2repository/com/android/volley/volley/1.0.1/volley-1.0.1.jar
        Required by:
            :analytics-android-integration-taplytics:unspecified
      > Could not find any matches for com.android.volley:volley:+ as no versions of com.android.volley:volley are available.
        Searched in the following locations:
            https://repo1.maven.org/maven2/com/android/volley/volley/maven-metadata.xml
            https://repo1.maven.org/maven2/com/android/volley/volley/
            https://github.com/taplytics/Taplytics-Android-SDK/raw/master/AndroidStudio/com/android/volley/volley/maven-metadata.xml
            https://github.com/taplytics/Taplytics-Android-SDK/raw/master/AndroidStudio/com/android/volley/volley/
            file:/Users/prateek/android-sdk/extras/android/m2repository/com/android/volley/volley/maven-metadata.xml
            file:/Users/prateek/android-sdk/extras/android/m2repository/com/android/volley/volley/
            file:/Users/prateek/android-sdk/extras/google/m2repository/com/android/volley/volley/maven-metadata.xml
            file:/Users/prateek/android-sdk/extras/google/m2repository/com/android/volley/volley/
        Required by:
            :analytics-android-integration-taplytics:unspecified > com.taplytics.sdk:taplytics:1.9.6
   > Cannot access first() element from an empty List

```

**Comment by VicV** [comment]
@f2prateek yeah looks like I'm a fool and thought 1.0.1 was a thing.

**Comment by VicV** [comment]
@f2prateek let me know if 1.0.0 works out.

**Comment by VicV** [comment]
@f2prateek anything else I need to do?

**Comment by f2prateek** [comment]
Yes this change does not build — the CI is still failing https://circleci.com/gh/segment-integrations/analytics-android-integration-taplytics/61 and it is locally failing for me with the same error as well.

**Comment by VicV** [comment]
🙌 🙌 🙌

**Comment by f2prateek** [comment]
Thanks!

**Pull request merged**